block-BHaaaf/index.html

```diff
@@ -0,0 +1,28 @@
+<!-- 
+  ### Create a simple page with an external stylesheet.
+
+  TODO:
+    - Create a simple page `(index.html)` with one heading following with a long paragraph.
+    - Create an external stylesheet `(style.css)` and change the color and font size of the heading and paragraph.
+    - Also, change the background color of the body.
+    - Use different selectors to select different elements.
+-->
+
```

Comment:
There must be a standard HTML document, starting with doctype consisting of the head tag also.